UI improvements

thinkh opened the pull request:
**prerequisites**: 
 * [x] branch is up-to-date with the branch to be merged with, i.e. develop
 * [x] build is successful
 * [x] code is cleaned up and formatted 


### Summary

* Use lighter background color on hover over column header
* Rename `Mapping / Scaling Type` to `Scaling` in mapping dialog
* Sort visualizations in ChangeRendererDialog by label
* Increase spacing for change renderer groups
* Use slightly darker primary action in dialogs
* Increase bottom space for histogram in side panel 
* Change placeholder text for search input
* Decrease font-size for side panel header
* Center row renderer content
* Modify checkbox label text for missing values
* Modify style for selected items indicator in side panel
* Modify style of side panel collapse toggle button
* Center side panel header text
* Rename `Stratify` to `Group by`

sgratzl commented:
starting with the upcoming release there is a new prerequisite for PR: 

* [ ] tested with Firefox 52, Firefox 57+, Chrome 64+, MS Edge 16+

sgratzl commented:
notes: 

![image](https://user-images.githubusercontent.com/4129778/42102776-4c55766a-7bc7-11e8-97aa-d2f9c11aef0a.png)

![image](https://user-images.githubusercontent.com/4129778/42102859-8460f2c8-7bc7-11e8-8f65-e22b20411e91.png)


new side collapser is overlapping with the add column chooser

![image](https://user-images.githubusercontent.com/4129778/42102816-66a97aac-7bc7-11e8-97b4-0c4d25f00fbf.png)

why so much space after the checkbox?

> Change placeholder text for search input

but it is not a search operation (different action) but actually filtering the data, thus is the label misleading right now. 

> Center row renderer content

do you know how "expensive" in terms of layout styling this is?

thinkh commented:
>  tested with Firefox 52, Firefox 57+, Chrome 64+, MS Edge 16+

Which tool do you use to test all these browsers? I don't have all these browsers installed on my machine...

thinkh commented:
I addressed your styling comments in the last commits. I moved most of the styles then to the lineup_app (branch taggle_user_study).

> do you know how "expensive" in terms of layout styling this is?
No, I'm not aware of it. Can you please explain/show it?

sgratzl commented on src/renderer/StringCellRenderer.ts:
```diff
@@ -1,10 +1,10 @@
-import {IDataRow, IGroup} from '../model';
+import { IDataRow, IGroup } from '../model';
```
why the formatting, which should even be disabled with the included `.vscode` settings?

thinkh replied:
You are right. The problem was that I did not open the lineupjs directory as VS Code root, but the parent directory. There were no `.vscode` settings available. I fixed the import with the last commit.
